src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<Figures> figuresList = new ArrayList<>();
```

Comment:
Use abstraction instead implementation where it possible
```suggestion
        List<Figures> figuresList = new ArrayList<>();
```